### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#7954] [Help]
Automation is not working for me after update.  My automations that were working perfectly fine still trigger but not sending emails.  I have tested a simple automation with just a delay (thinking the problem is with emails), still does not work.  Does not work when I test the automation within the automation design screen.  Even after automations are triggered in the published version, the automations are not showing up on the automation history.  In fact, all my automation history is not showing.  Anyone help please.

**Comment by melohagan** [comment]
Hey @ihabmzaky 

Is this in cloud? Could you please provide your tenant id please? (e.g. `<my-id>.budibase.app`

**Reply by ihabmzaky** [reply]
It just made sense now, it is on Budibase cloud hence it stopped because of the new pricing plan by Bubibase.  It just stopped earlier than the release of 2.0

**Reply by melohagan** [reply]
Oh okay. Did you hit quota limits then?

**Reply by ihabmzaky** [reply]
apparently I did!  Is there an automation quota for the open source self hosted as well?  It is not clear

**Reply by melohagan** [reply]
![Screenshot 2022-09-28 at 09 53 28](https://user-images.githubusercontent.com/101575380/192735281-12f4475e-36ab-47d5-acab-8183aa71cc05.png)

If you click on Usage you'll be able to see any limit you've hit
